CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Releases
 
+## Version 0.14.0
+
+Date: 2022-09-30
+
+This release focuses on three main themes, support for running Panel entirely in the browser using WebAssembly (via Pyodide and PyScript), quality of life improvements to make it easier to build responsive and performant applications and finally improvements to static typing and docstrings to enhance the developer experience.
+
+Many, many thanks to everyone who filed issues or contributed to this release. In particular we would like to thank @janimo, @xavArtley, @thuydotm, @jmosbacher, @dmarx, 2WoLpH, @ipopa144, @sdc50 and the core team consisting of @philippjfr, @Hoxbro, @maximlt and @MarcSkovMadsen.
+
+### Features
+
+- Add support for converting Panel apps to pyscript/pyodide (#3817, #3830, #3851, #3856, #3857, #3858, #3860, #3861, #3863, #3864, #3868, #3878)
+- Manage our own CDN (#3867, #3870)
+- Add `pn.widget` helper function (#1826)
```

Comment:
```suggestion
- Add `pn.widget` helper function (#3589, #1826)
```
(1826 is just the issue, not the PR, though including it does make sense because the PR doesn't explain what it's for.)

Has this subsequently been documented? If so the doc PR should also be linked here. If not, need to document it!